site/js/modules/webui/server.js

```diff
@@ -55,19 +57,36 @@ function handlerCallAPI(path, params, context, container) {
     ;
 }
 
-function render(endpoints, selectedEndpoint, context, container) {
+function render(endpoints, selectedEndpoint, params, context, container) {
+    let description = undefined;
+    if (selectedEndpoint in endpoints) {
+        description = endpoints[selectedEndpoint].description;
+    } else {
+        selectedEndpoint = undefined;
+    }
+
     let selector = renderSelector(endpoints, selectedEndpoint);
```

Comment:
Ideally, rendering this selector will share code with how we render other inputs.